### PR TITLE
chore(Dockerfile,__init__.py): DEIS_RELEASE -> WORKFLOW_RELEASE

### PR DESCRIPTION
**Pull request by vdice**
# Summary of Changes

Updated `DEIS_RELEASE` to be `WORKFLOW_RELEASE` as done similarly elsewhere (see https://github.com/deis/workflow/pull/193)
TODO
- [ ] similar PR in https://github.com/deis/example-go/blob/master/web.go#L24-L27

# Pull Request Hygiene TODOs

Please make sure the below checklist is complete.

- [x] Your pull request is concise and to the point (make another PR for refactoring nearby code)
- [x] Your commits are squashed into logical units of work
- [x] Your commits follow the [commit style guidelines](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide)

### CODE REVIEW

File: rootfs/scheduler/__init__.py [hunk 1]
```diff
@@ -142,7 +142,7 @@
                 "value":"$id"
             },
             {
-                "name":"DEIS_RELEASE",
+                "name":"WORKFLOW_RELEASE",
```
Comment (helgi) [inline rootfs/scheduler/__init__.py]: Does anyone know why we inject this into the RCs? It uses `$appversion` and well... that's the application release version - Can we remove it?

Either way, I don't think this should be renamed if it is not being used for release purposes

Comment (bacongobbler) [inline rootfs/scheduler/__init__.py]: it's used so that the application can gain knowledge of its own release information. It's important!

Comment (bacongobbler) [inline rootfs/scheduler/__init__.py]: we should update example-go to use WORKFLOW_RELEASE: https://github.com/deis/example-go/blob/master/web.go#L24-L27

Comment (helgi) [inline rootfs/scheduler/__init__.py]: Ah, right, okay if all of those get updated then we can make this change :)

Comment (vdice) [inline rootfs/scheduler/__init__.py]: thanks @bacongobbler & @helgi ; example-go PR here https://github.com/deis/example-go/pull/14

**Pull request merged**